packages/sdk/src/sdk.ts

```diff
@@ -365,31 +377,80 @@ export class NxtpSdk {
     }
 
     const inbox = generateMessagingInbox();
-    const receivedResponsePromise = new Promise<AuctionResponse>((res, rej) => {
+
+    let AuctionBids: AuctionResponse[] = [];
```

Comment:
Don't capitalize variable name like this.

Reply:
done